src/core/server/base.js

```diff
@@ -579,6 +579,17 @@ export function runServer({
                 `[apiVersion:${config.get('apiVersion')}]`,
               ].join(' '),
             );
+
+            const strictModeEnabled = convertBoolean(
+              config.get('enableStrictMode'),
+            );
+
+            if (strictModeEnabled) {
+              log.info(
+                `🔥  StrictMode enabled, checkout this link https://github.com/mozilla/addons-frontend/issues/6424.`,
+              );
+            }
```

Comment:
Could you put that in `src/core/client/base.js` instead please? I think we do not need the flame (although I like it).

Could you put this message:

```
StrictMode is enabled, which  causes double redux action dispatching. See: https://github.com/mozilla/addons-frontend/issues/6424
```